### PR TITLE
Nazar/feature/plant detail info

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: src/components/PlantDetailInfo/PlantDetailInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { sizes } from '@src/styling/sizes';
+import { styleSheet } from './PlantDetailInfo.style';
+
+type PlantDetailInfoProps = {
+	title?: string;
+	subTitle: string;
+	SvgIcon: FunctionComponent<SVGAttributes<SVGElement>>;
+};
+
+const SIZE = 48;
```
Comment (marynag) [inline src/components/PlantDetailInfo/PlantDetailInfo.tsx:13]: use size from theme: theme.sizes.xxl + theme.sizes.lg

Comment (Taraslysak) [reply src/components/PlantDetailInfo/PlantDetailInfo.tsx:13]: +1

File: src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { PlantSize } from './constants/plant';
+
+export const getSizeKey = (maxSize: number): string => {
+	if (maxSize > 100) {
+		return PlantSize.large;
+	} else if (maxSize > 50) {
```
Comment (Taraslysak) [inline src/utils.ts:6]: you can remove `else` here and just keep `if`

File: src/constants/plant.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export enum PlantSize {
+	small = 'small',
```
Comment (Taraslysak) [inline src/constants/plant.ts:2]: It is nicer to have enum keys in UPPER_CASE

**Review by Taraslysak — APPROVED** [review]
Good Job!

**Pull request merged**